packages/graph/src/-private/-diff.ts

```diff
@@ -52,9 +54,17 @@ function _deprecatedCompare<T>(
         adv = true;
 
         if (!prevSet.has(member)) {
-          changed = true;
-          added.add(member);
-          onAdd(member);
+          if (!prevSet.has(member)) {
```

Comment:
`if (!prevSet.has(member)) {` is defined twice, is this intended?
Sorry for the out-of-the-blue review, stumbled upon this PR and thought I should ask / mention it.

Reply:
Def not intentional. Thanks for the catch.